lib/api/src/js/export.rs

```diff
@@ -45,6 +52,52 @@ impl VMMemory {
     pub(crate) fn try_clone(&self) -> Option<VMMemory> {
         Some(self.clone())
     }
+
+    /// Copies this memory to a new memory
+    pub fn fork(&self) -> Result<VMMemory, wasmer_types::MemoryError> {
```

Comment:
Let's use clone. I think is a more accurate naming

Reply:
I'm not in favor of `clone` either, as the operation is quite different from the `Clone` trait.

Reply:
Renamed to `duplicate()`.